BitwardenShared/UI/Auth/IntroCarousel/IntroCarouselState.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftUI
+
+// MARK: - IntroCarouselState
+
+/// An object that defines the current state of a `IntroCarouselView`.
+///
+struct IntroCarouselState: Equatable {
+    // MARK: Types
+
+    /// A model representing the data to display on a single page in the carousel.
+    ///
+    struct CarouselPage: Equatable, Identifiable {
+        // MARK:
+
+        /// A unique identifier of the page.
+        let id: String = UUID().uuidString
```

Comment:
will this cause issues with swiftui updating unnecessarily?

Reply:
I don't think it should, `pages` is a constant within the state. It would only be an issue if we were reinitializing the pages, but this content should be static.

Reply:
i see!  makes sense, thanks!